scribble-lib/help/help.rkt

```diff
@@ -1,4 +1,4 @@
-#lang scheme/base
+#lang racket/base
 
 (require "search.rkt" scheme/cmdline scheme/list scheme/string
```

Comment:
Since you are touching these files already, I think it's worth converting these libraries (`scheme/cmdline`, `scheme/list`, etc.) to the corresponding `racket` libraries too.

Reply:
Agreed, but I want to do that in a separate pull request so it's easier to make sure this first step doesn't break anything.